config/manager/csi-images.yaml

```diff
@@ -1,20 +1,20 @@
 ---
```

Comment:
it'd be better to add new stanza for version: v4.16

Reply:
Should we add stanzas for 4.14, 4.15, and 4.16 as we are supporting these 3 versions, and remove 4.11 and 4.12?

Reply:
ok, you can remove everything below and including 4.13, add 4.14, 4.15, 4.16 stanzas it'd be good if we can refer to sidecars used in those timeframes (stretch but not mandatory).